examples/2d/sprite_sheet.rs

```diff
@@ -45,9 +52,11 @@ fn setup(
     commands.spawn((
         SpriteSheetBundle {
             texture_atlas: texture_atlas_handle,
+            sprite: TextureAtlasSprite::new(1),
```

Comment:
This `1` literal is repeated below, and these values must remain in sync.

Maybe extract `AnimationIndices` into a variable and reference its `first` field here?

Reply:
Good idea, done